Exercise_A/index.html

```diff
@@ -17,11 +17,132 @@
 
       Refer to the Practice directory
     -->
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">  
+    <!-- JavaScript Bundle with Popper -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+    <link href="./assets/css/style.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/75a20021a3.js" crossorigin="anonymous"></script>
   </head>
 
-  <!-- Nav bar setup -->
 
-  <body data-spy="scroll" data-target=".navbar">
+  <body>
+    <nav class="navbar fixed-top scrolling-navbar navbar-light bg-light navbar-expand-sm">
+      <a href="#" class="navbar-brand">THE ONLY BOOTSTRAP PAGE</a>
+      <button class="navbar-toggler ms-auto" data-toggle="collapse" data-target="#navbarMenu">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarMenu">
+        <ul class="navbar-nav ms-auto">
+          <li class="nav-item"><a href='#About' class="nav-link">About</a></li>
+          <li class="nav-item"><a href='#Four' class="nav-link">Four Icons</a></li>
+          <li class="nav-item"><a href='#Bullshit' class="nav-link">Bullshit</a></li>
+          <li class="nav-item"><a href='#Call' class="nav-link">Call Me</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="bg-image p-5" style="background-image: url(./assets/images/header.jpg); height:40rem; background-size: cover; background-repeat: no-repeat; background-position: bottom;">
+      <div class="container">
+        <div class="d-flex justify-content-center">
+          <div class="align-content-center text-center">
+            <h1 class='text-center' style="width:70%; margin-left: 15%; margin-top:10%; color:white;"><strong>HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER</strong></h1>
+            <hr class="align-content-center">
+            <p class="lead text-center" style="color:white;"> Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!</p>
+            <button type="button" class="btn btn-warning">This Button Makes The Page Scrolls</button>

```

Comment:
I think you may have forgotten to add the scroll part for the button, as when this is clicked the page doesn't scroll down. You can wrap this inside a link and give that link an ID so that it then scrolls to the bottom. But also remember you would need to give an element the same ID